doc/config.yml

```
@@ -1,6 +1,7 @@
 Content:
     - ${ROOT_DIR}/doc/content
     - ${MOOSE_DIR}/framework/doc/content
+    - ${MOOSE_DIR}/modules/heat_conduction/doc/content
```

```suggestion
    - ${MOOSE_DIR}/modules/heat_conduction/doc/content
    - ${MOOSE_DIR}/modules/ray_tracing/doc/content      # pulled in by heat_conduction
```

You'll need this for the main documentation build (heat conduction has ray tracing as a dependency). I would also explicitly list `RAY_TRACING := yes` in the Makefile for the reminder to the developers that this dependency exists.